config.nims

```diff
@@ -25,7 +25,7 @@ case getEnv "ENV":
   of "DEV":
     --assertions:on
     --checks:on
-    --colors:on
+    # --colors:on # breaks vscode runner?
```

Comment:
none of the color codes are interpreted

Reply:
find which repo this belongs to and create a bug
not sure if its a nimlang issue or a vscode runner issue

Reply:
if we dont set `colors: on` the colors are interpreted, maybe just ignore this and move on with life